tests/testthat/test-misc.R

```diff
@@ -1,11 +1,68 @@
-test_that("keygen returns error when type not in c('rsa', 'dsa'", {
-  expect_error(sship_keygen(directory = tempdir(), type = "abc"))
+defaultW <- getOption("warn")
+
+test_that("keygen returns error when type not known", {
+  expect_error(keygen(directory = tempdir(), type = "abc"))
+})
+
+test_that("an rsa key pair can be made", {
+  expect_message(keygen(directory = tempdir()))
 })
 
 test_that("a dsa key pair can be made", {
-  expect_message(sship_keygen(directory = tempdir(), type = "dsa"))
+  expect_warning(
+    keygen(directory = tempdir(), type = "dsa", overwrite_existing = TRUE)
+  )
+  options(warn = -1)
```

Comment:
Kan med fordel flytta `defaultW` ned *her* (som er her han blir brukt), og heller skriva han som

    defaultW <- options(warn = -1)

(`options()` returnerer originalverdien/-ane.) Så kan du rett nedanfor skriva

    options(defaultW)

for å tilbakestilla.

Reply:
Se https://github.com/Rapporteket/sship/pull/34/commits/8c7b6612b7f6f2f1626452deb2e9695f76103e4d